src/js/experimental/media-chrome-selectmenu.js

```diff
@@ -198,6 +197,8 @@ class MediaChromeSelectMenu extends globalThis.HTMLElement {
   #hide() {
     if (this.#listboxSlot.hidden) return;
 
+    unobserveResize(getBoundsElement(this), this.#updateMenuPosition);
```

Comment:
question(non-blocking): Any reason for the reorder here?

Reply:
yes, if the listbox is hidden we don't want the updateMenuPosition to muddle the transition in case the element gets a resize observe event